packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -62,78 +62,78 @@
   min-width: var(--LabelMinWidth);
 
   &:not(:only-child) {
-    margin-inline-start: var(--LabelSpaceMarginInlineStart);
+    margin-inline-start: var(--LabelMarginInlineStart);
   }
 }
 
 /* Variants */
 
 .secondaryVariant {
-  border-color: var(--ColorBorderSecondary);
-  background-color: var(--ColorBackgroundSecondary);
-  color: var(--ColorTextSecondary);
+  border-color: var(--SecondaryBorderColor);
+  background-color: var(--SecondaryBackgroundColor);
+  color: var(--SecondaryTextColor);
 
   &:hover {
-    border-color: var(--ColorBorderSecondaryHover);
-    background-color: var(--ColorBackgroundSecondaryHover);
-    color: var(--ColorTextSecondaryHover);
+    border-color: var(--SecondaryHoverBorderColor);
+    background-color: var(--SecondaryHoverBackgroundColor);
+    color: var(--SecondaryHoverTextColor);
   }
 
   &:focus {
-    border-color: var(--ColorBorderSecondaryFocus);
-    background-color: var(--ColorBackgroundSecondaryFocus);
-    color: var(--ColorTextSecondaryFocus);
+    border-color: var(--SecondaryFocusBorderColor);
+    background-color: var(--SecondaryFocusBackgroundColor);
+    color: var(--SecondaryFocusTextColor);
   }
 
   &:disabled {
-    border-color: var(--ColorBorderSecondaryDisabled);
-    background-color: var(--ColorBackgroundSecondaryDisabled);
-    color: var(--ColorTextSecondaryDisabled);
+    border-color: var(--SecondaryDisabledBorderColor);
+    background-color: var(--SecondaryDisabledBackgroundColor);
+    color: var(--SecondaryDisabledTextColor);
   }
 }
 
 .dangerVariant {
-  background-color: var(--ColorBackgroundDanger);
+  background-color: var(--DangerBackgroundColor);
 
   &:hover {
-    border-color: var(--ColorBorderDangerHover);
-    background-color: var(--ColorBackgroundDangerHover);
+    border-color: var(--DangerHoverBorderColor);
+    background-color: var(--DangerHoverBackgroundColor);
   }
 
   &:focus {
-    @include outline(var(--ColorFocusOutlineDanger), var(--FocusOutlineWidth));
+    @include outline(var(--DangerFocusOutlineColor), var(--FocusOutlineWidth));
 
-    background-color: var(--ColorBackgroundDangerFocus);
+    background-color: var(--DangerFocusBackgroundColor);
   }
 
   &:disabled {
-    border-color: var(--ColorBorderDangerDisabled);
-    background-color: var(--ColorBackgroundDangerDisabled);
+    border-color: var(--DangerDisabledBorderColor);
+    background-color: var(--DangerDisabledBackgroundColor);
   }
 }
 
 .dismissVariant,
 .dismissInvertedVariant {
-  padding-block: calc(var(--SpacePaddingBlockDismiss) * 0.5);
-  padding-inline: calc(var(--SpacePaddingInlineDismiss) * 0.5);
+  padding-block: calc(var(--DismissPaddingBlock) * 0.5);
+  padding-inline: calc(var(--DismissPaddingInline) * 0.5);
   border: 0;
-  background-color: var(--ColorBackgroundDismiss);
-  color: var(--ColorTextDismiss);
-  line-height: var(--FontLineHeightDismiss);
+  background-color: var(--DismissBackgroundColor);
+  color: var(--DismissTextColor);
```

Comment:
`DismissColor` my understanding is we don't prefix with groupings like Text any more